applications/emocam/emocam1.js

```diff
@@ -222,18 +232,6 @@ var webWindow;
         }
     }
 
-    function setEmotion(currentEmotion) {
```

Comment:
I'm curious how removing this function will change the blendshape behavior.  Is this something you added earlier and are now removing or did you find it created some issues?

Reply:
That function was not used anywhere